libs/utils/address.go

```diff
@@ -30,14 +30,11 @@ func ValidateAddr(addr string) (string, error) {
 	}
 
 	if ip := net.ParseIP(addr); ip == nil {
```

Comment:
nice to have would be to return fast in case ip != nil. It would also help to return fast resolved addr

Reply:
ok nice suggestion